presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -571,7 +571,7 @@ Procedures
   * ``DROP``: drop any partitions that exist in the metastore, but not on the file system.
   * ``FULL``: perform both ``ADD`` and ``DROP``.
 
-  The ``case_sensitive`` argument is optional. The default value is ``true`` for compatibility
+  The default value for the optional ``case_sensitive`` argument is ``true`` for compatibility
```

Comment:
I think the original is clearer, since it starts with the argument name and doesn't split the sentence. The new version results in a longer sentence, and makes the "which expects" part unclear as to what it applies to.